onyo/cli/get.py

```diff
@@ -121,6 +122,13 @@
 }
 
 epilog_get = r"""
+.. rubric:: Exit Status
+
+The exit status is ``0`` if at least one result is found, ``1`` if there are no
```

Comment:
It surprises me that you would not use a list to explain 0, 1, 2 :)

Reply:
Yeah, I copied `grep`'s manpage and worked from there. I think it's clear enough as it is, and uses less vertical space.